src/main/kotlin/com/terraformation/backend/i18n/Messages.kt

```diff
@@ -70,27 +70,15 @@ class Messages {
 
   fun applicationModulesIncomplete() = getMessage("application.failure.modulesIncomplete")
 
-  fun applicationStatusUpdatedNotifcation(status: ExternalApplicationStatus): NotificationMessage =
-      when (status) {
-        ExternalApplicationStatus.Accepted ->
-            NotificationMessage(
-                getMessage("notification.application.accepted.app.title"),
-                getMessage("notification.application.accepted.app.body"),
-            )
-        ExternalApplicationStatus.NotAccepted ->
-            NotificationMessage(
-                getMessage("notification.application.notAccepted.app.title"),
-                getMessage("notification.application.notAccepted.app.body"),
-            )
-        ExternalApplicationStatus.Waitlist ->
-            NotificationMessage(
-                getMessage("notification.application.waitlist.app.title"),
-                getMessage("notification.application.waitlist.app.body"),
-            )
-        else ->
-            throw IllegalArgumentException(
-                "Application status $status does not require notifications")
-      }
+  fun applicationSubmittedNotification(
+      organizationName: String,
+      timeSubmitted: Instant
+  ): NotificationMessage =
+      NotificationMessage(
+          getMessage("notification.application.submitted.app.title"),
+          getMessage(
+              "notification.application.submitted.app.body", organizationName, timeSubmitted),
```

Comment:
The time doesn't appear to be used. The message only has a `{0}` placeholder, no `{1}`.